content/nb_collectives_intro.tex

```diff
@@ -1,7 +1,8 @@
 An \openshmem nonblocking collective operation, like a blocking collective
 operation, is a group communication operation among the
 participants of the team. All participants of the team are required to call the
-collective operation.
+collective operation and the collective operations must occur in the same
+program order across all PEs in the team.
```

Comment:
clarify initiation order vs execution order.

initiate in the same order across all PEs, executed in any order.